sklearn/pipeline.py

```diff
@@ -774,12 +818,8 @@ def fit(self, X, y=None):
         self : FeatureUnion
             This estimator
         """
-        self.transformer_list = list(self.transformer_list)
-        self._validate_transformers()
-        transformers = Parallel(n_jobs=self.n_jobs)(
-            delayed(_fit_one_transformer)(trans, X, y)
-            for _, trans, _ in self._iter())
-        self._update_transformer_list(transformers)
+        fit_params = {}  # XXX: this should probably be added to method
```

Comment:
?

Reply:
I'll remove this comment. Adding `fit_params` to `FeatureUnion` is discussed: https://github.com/scikit-learn/scikit-learn/issues/7136 which leads down the road to `sample_props`.